cfme/fixtures/cli.py

```diff
@@ -88,12 +90,20 @@ def appliance_with_preset_time(temp_appliance_preconfig_funcscope):
     return temp_appliance_preconfig_funcscope
 
 
-@pytest.yield_fixture()
-def ipa_crud(configured_appliance, ipa_creds):
-    configured_appliance.appliance_console_cli.configure_ipa(ipa_creds['ipaserver'],
-        ipa_creds['username'], ipa_creds['password'], ipa_creds['domain'], ipa_creds['realm'])
+@pytest.fixture()
+def ipa_crud():
+    try:
+        ipa_keys = [key
+                    for key, yaml in iteritems(auth_data.auth_providers)
+                    if yaml.type == authutil.FreeIPAAuthProvider.auth_type]
+        ipa_provider = authutil.get_auth_crud(ipa_keys[0])
+    except AttributeError:
+        pytest.mark.uncollect('Unable to parse auth_data.yaml for freeipa server')
```

Comment:
**REQUIRED:** Shouldn't this skip? The test is valid, we just don't have the data for it?

Reply:
Agreed.